trestle/core/commands/author/profile.py

```diff
@@ -279,9 +286,10 @@ def assemble_profile(
             md_name: The name of the directory containing the markdown control files for the profile
             new_profile_name: The name of the new json profile.  It can be the same as original to overwrite
             set_parameters: Use the parameters in the yaml header to specify values for setparameters in the profile
-            regenerate: whether to regenerate the uuid's in the profile
-            version: version for the assembled profile
-            required_sections: list of required sections in the assembled profile as comma-separated short names
+            regenerate: Whether to regenerate the uuid's in the profile
+            version: Optional version for the assembled profile
+            required_sections: Optional List of required sections in assembled profile as comma-separated short names
+            allowed_sections: Optional list of section short names that are allowed
 
```

Comment:
Is this better -
allowed_sections: Optional list of sections that are allowed in assembled profile as comma separated short names

Reply:
Yes that is better.  Changed.